docs-site/docs/user/project/samples/sample-metadata.md

```diff
@@ -4,10 +4,160 @@ id: sample-metadata
-- Add metadata
-- Update metadata
-- Import metadata
-- Delete metadata
- -->
+Metadata can be added to samples to give them any additional information required by users.
+
+A sample cannot have metadata with the same key, and metadata added by an analysis cannot be overwritten by a user.
+
+## View Metadata
+
+1. From the left sidebar, select **Projects**
+2. Select the project
+3. From the left sidebar, select **Samples**
+4. Click the toggle beside **Metadata**
+
+The samples table will update with all the samples' metadata.
+
+To view individual sample metadata:
+
+5. Select the sample
+6. Click the **Metadata** tab
+
+## Add Metadata
+
+Prerequisites:
+
+- You must have at least a **Maintainer** role for the sample's project
+- A sample added to the project
+
+1. From the left sidebar, select **Projects**
+2. Select the project
+3. From the left sidebar, select **Samples**
+4. Select the sample
+5. Click the **Metadata** tab
+6. Click **Add Metadata**
+
+A dialog will pop-up where you can add new metadata.
+
+## Update Metadata
+
+Prerequisites:
+
+- You must have at least a **Maintainer** role for the sample's project
+- A sample with existing metadata added to the project
+
+1. From the left sidebar, select **Projects**
+2. Select the project
+3. From the left sidebar, select **Samples**
+4. Select the sample
+5. Click the **Metadata** tab
+6. Click the **Update** link of the metadata you'd like to update
+
+A dialog will pop-up where you can update the metadata.
+
+## Learn About Importing Metadata
+
+Importing metadata allows you to add multiple metadata fields to multiple samples all at once. This requires a spreadsheet in .csv, .xls, or .xlsx format.
+
+This is an example of the expected spreadsheet format:
+
+  | Sample Name | MetadataField1 | MetadataField2 | MetadataField3 |
+  | :---------- | :------------- | :------------- | :------------- |
+  | Sample 1    | value1         | value2         | value3         |
+  | Sample 2    | value4         | value5         | value6         |
+
+The following metadata will be added:
+
+  - Sample 1:
+
+    | Key            | Value  |
+    | :------------- | :----- |
+    | MetadataField1 | value1 |
+    | MetadataField2 | value2 |
+    | MetadataField3 | value3 |
+
+  - Sample 2:
+
+    | Key            | Value  |
+    | :------------- | :----- |
+    | MetadataField1 | value4 |
+    | MetadataField2 | value5 |
+    | MetadataField3 | value6 |
+
+When creating the spreadsheet, you are required to have a column that contains a sample identifier. The identifier is case-sensitive and can contain either the sample names or IDs.
+
+**An important note:** When importing a metadata spreadsheet, you will be asked if you'd like to **Ignore empty values**. If this is **selected**, any metadata fields without an associated value will not be added to the sample. However, if this **not selected**, any samples with the metadata key and empty value will be **deleted**.
```

Comment:
**nitpick (blocking):** Can you rephrase the following:

`If this is **selected**, any metadata fields without an associated value will not be added to the sample.`

to:

`If this is **selected**, any metadata fields without an associated value will be ignored and those metadata keys will not be removed from the sample if present.`

Reply:
Replaced in d55b07c09b33c3914f3f21c5a0f2ee10aa7e2734